From 39d41be24e718c928a9ca7e21b103913f78ea6ac Mon Sep 17 00:00:00 2001
From: George G <kreach3r@users.noreply.github.com>
Date: Wed, 8 Feb 2017 17:22:44 +0200
Subject: [PATCH 04/13] sepolicy: fix themed sounds

02-08 17:26:48.011 18259-18259/? W/SoundPoolThread: type=1400 audit(0.0:31): avc: denied { read } for path="/data/system/theme/audio/ui/Lock.ogg" dev="dm-0" ino=1006317 scontext=u:r:drmserver:s0 tcontext=u:object_r:theme_data_file:s0 tclass=file permissive=0

Change-Id: If96d784d4a79e7c7f7d21d191c2e0795c366e03a
---
 drmserver.te | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drmserver.te b/drmserver.te
index 9130e0b..6d3883f 100644
--- a/drmserver.te
+++ b/drmserver.te
@@ -54,3 +54,7 @@ allow drmserver drmserver_service:service_manager { add find };
 allow drmserver permission_service:service_manager find;
 
 selinux_check_access(drmserver)
+
+# Themed resources (i.e. composed icons)
+allow drmserver theme_data_file:dir r_dir_perms;
+allow drmserver theme_data_file:file r_file_perms;
-- 
2.9.4

